tests/suite/rename/rename-error-dupe.yaml

```diff
@@ -0,0 +1,12 @@
+script: zq -t "rename foo=goo" -
```

Comment:
This should be a ZQL test, not a script test.

Reply:
I made these script tests because they look at stderr, and a zql test fails due to exit status 1. Is there an example of a zql test that tests errors that I'm missing?

Reply:
There are.

```
$ git grep -l errorRE -- *.yaml
tests/suite/errors/nosemi.yaml
tests/suite/put/put-6.yaml
tests/suite/reducer/duplicate.yaml
```

Reply:
thanks!

Reply:
now i know why i had the recollection that zql tests didn't fully work for error checks: https://github.com/brimsec/zq/pull/1001